android/assets/jsons/translations/Dutch.properties

```diff
@@ -1525,10 +1494,8 @@ Choose a pantheon = Kies een pantheon
 Choose a Religion = Kies een religie
 Found Religion = Sticht religie
 Found Pantheon = Sticht pantheon
- # Requires translation!
-Reform Religion = 
- # Requires translation!
-Expand Pantheon = 
+Reform Religion = Herzie religie
```

Comment:
Ik zou het eerder 'reformeer religie' noemen

Reply:
Fair point, though I dislike the word `reformeer`. I think there are simpler words with the same meaning. I agree that `Herzie` doesn't quite do it, but then I would suggest `Hervorm` because it's simpler and more well-known.